core/domain/fs_domain.py

```diff
@@ -471,20 +471,47 @@ def isfile(self, filepath):
         except cloudstorage.NotFoundError:
             return False
 
-    def get(self, filepath, version=None, mode='r'):  # pylint: disable=unused-argument
-        """Raises NotImplementedError if the method is not implemented in the
-        derived classes.
+    def _get_file_data(self, filepath):
+        """Gets the content of the file stored in GCS
```

Comment:
End with period. Add new line below this one.

In general please self-review your PR and ensure that your code and docstrings conform to the existing style.

Reply:
Sorry, Will take care of it in future.